### PR TITLE
Enhance Poster Rendering

### PR DESCRIPTION
**Pull request by jishnurk**
@mhdzumair: Planning to make some more changes over the weekend as you suggested. Let's keep the PR live.

As a first step, replaced "IMDb" text with logo in Poster Rendering.

Example:
preview

## Summary by CodeRabbit

- **New Features**
	- Introduced a new sports category, "AEW" (All Elite Wrestling), with sections for background images, posters, and logos.
	- Enhanced presentation of IMDb rating on posters with improved logo display and alignment.
	- Directly displays the rating in the format "{imdb_rating}/10" without the "IMDb:" prefix.

- **User Interface Improvements**
	- Updated existing sports entries with new URLs for backgrounds, posters, and logos, enhancing visual content.
	- Adjusted layout for better visual appeal and clarity in displaying the IMDb logo and rating.

### CODE REVIEW

File: utils/poster.py [hunk 1]
```diff
@@ -89,27 +89,35 @@ def add_elements_to_poster(
 
     # Adding IMDb rating at the bottom left with a semi-transparent background
     if imdb_rating:
+        imdb_text = f" {imdb_rating}/10"
+        imdb_logo = Image.open("resources/images/imdb_logo.png")
```
[the remark this answers is not in the document]
Comment (mhdzumair) [reply utils/poster.py:93]: Images will be there as always.

**Comment by mhdzumair** [comment]
This is nice. BTW You don't have to follow every code review

File: utils/poster.py [hunk 2]
```diff
@@ -89,27 +89,35 @@ def add_elements_to_poster(
...
 
-        # Draw a semi-transparent rectangle behind the text for better visibility
+        # Resize IMDb Logo according to text height
+        aspect_ratio = imdb_logo.width / imdb_logo.height
+        imdb_logo = imdb_logo.resize((int(text_height * aspect_ratio), text_height))
+
+        # Draw a semi-transparent rectangle behind the logo and rating for better visibility
         rectangle_x0 = margin
-        rectangle_x1 = rectangle_x0 + text_width + (2 * padding)
+        rectangle_x1 = rectangle_x0 + imdb_logo.width + text_width + (2 * padding)
```
[the remark this answers is not in the document]
Comment (jishnurk) [reply utils/poster.py:107]: Explicit check is not required. In most cases, width of rectangle box will be less than half the poster width.

File: utils/poster.py [hunk 3]
```diff
@@ -89,27 +89,35 @@ def add_elements_to_poster(
...
         rectangle_y0 = image.height - margin - text_height - (2 * padding)
         rectangle_y1 = image.height - margin
         draw.rounded_rectangle(
             (rectangle_x0, rectangle_y0, rectangle_x1, rectangle_y1),
             fill=(0, 0, 0, 176), radius=8
         )
 
-        # Now draw the text
+        # Place the IMDb Logo
+        image.paste(imdb_logo, (rectangle_x0 + padding, rectangle_y0 + padding), imdb_logo)
```
[the remark this answers is not in the document]
Comment (jishnurk) [reply utils/poster.py:116]: `imdb_logo` is created and resized from a static file.

File: utils/poster.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jishnurk) [reply utils/poster.py:107]: I would leave it as it is for now.

[the remark this answers is not in the document]
Comment (jishnurk) [reply utils/poster.py:107]: I would leave it as it is for now.

File: utils/poster.py [hunk 4]
```diff
@@ -89,27 +89,35 @@ def add_elements_to_poster(
...
+
+        # Now draw the rating text
         draw.text(
-            (rectangle_x0 + padding, rectangle_y0), imdb_text, font=font, fill="#F5C518"
+            (rectangle_x0 + padding + imdb_logo.width, rectangle_y0), imdb_text, font=font, fill="#F5C518"
```
[the remark this answers is not in the document]
Comment (jishnurk) [reply utils/poster.py:120]: Explicit check is not required.

**Comment by mhdzumair** [comment]
BTW if you need to ask anything You can connect me via TG https://t.me/mediafusion_stremio_addon

**Comment by mhdzumair** [comment]
Mark this as draft and whenever you're done. LMK.

**Pull request merged**